generated/nidigital/session.py

```diff
@@ -2676,6 +2676,85 @@ def export_signal(self, signal, signal_identifier, output_terminal):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, timeout):
+        # This is slightly modified codegen from the function
```

Comment:
The reason we have this hand-coded implementation is that "the codegenerator does not support ivi-dance with two values that are multiplied together".

The reason should be documented here.
It's a non-trival amount of code to hand-maintain. It worries me because the fix #1092 will likely affect this sort of code and we are at risk of not updating this hand maintained copy.

Reply:
Already added.